src/onboarding2/Onboarding/Onboarding.js

```diff
@@ -1,15 +1,22 @@
-import React, { useCallback, useState } from 'react'
+import React, { useCallback, useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
 import { useTheme, BREAKPOINTS } from '@aragon/ui'
+import { resolveEnsDomain } from '../../aragonjs-wrapper'
 import ConnectModal from '../../components/ConnectModal/ConnectModal'
+import { EthereumAddressType } from '../../prop-types'
 import { saveTemplateState } from '../create-utils'
+import templates from '../../templates'
 import Welcome from '../Welcome/Welcome'
 import Create from '../Create/Create'
 import OnboardingTopBar from './OnboardingTopBar'
 
-function Onboarding({ status, selectorNetworks }) {
+function Onboarding({ account, status, selectorNetworks, walletWeb3, web3 }) {
   const theme = useTheme()
   const [connectModalOpened, setConnectModalOpened] = useState(false)
+  const [availableTemplates, setAvailableTemplates] = useState([])
```

Comment:
@bpierre I implemented what we discussed earlier about doing the prefetch ENS check on each template before moving to `Create`.

The fetching state is sent to `Welcome`, but its UI needs to be implemented.